fmoe/layers.py

```diff
@@ -196,14 +204,33 @@ def forward(self, inp):
 
         gate_top_k_idx, gate_score = self.gate(inp)
 
-        x = _fmoe_general_global_forward(
-            inp, 
+        # delete masked tensors
+        if self.mask != None and self.mask_dict != None:
```

Comment:
plz use `is not None` instead of `!= None`

Reply:
fixed in the new commit